### PR TITLE
Refactor appsTests and capabilitiesTest

### PR DESCRIPTION
**Pull request by dpakach**
### Refactor appsTests and capabilitiesTest
1. Add pactLog step in CI for showing pactio log files
2. Remove callbacks from the test hook functions (beforeAll, afterAll, beforeEach, afterEach)
3. use createOwncloud() function from `pacthelper` to create new instance of owncloud sdk

Related https://github.com/owncloud/owncloud-sdk/issues/581

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -11,7 +11,12 @@ config = {
 }
 
 def main(ctx):
-    return [ fullBuild(), testWithinSubFolder(), publish() ]
+    pipelines = []
+    for i in range(10):
+        pipelines.append(fullBuild(i))
+        pipelines.append(testWithinSubFolder(i))
+    # return [ fullBuild(), testWithinSubFolder(), publish() ]
```
Comment (individual-it) [inline .drone.star:18]: extra comment?

**Review by individual-it — CHANGES_REQUESTED** [review]
please write a couple of lines about what this PR should do, I see changes to drone but also changes related to promises

File: tests/appsTest.js [hunk 2]
```diff
@@ -131,35 +131,16 @@ describe('Main: Currently testing apps management,', function () {
...
     }
 
-    Promise.all(promises).then(done, done.fail)
+    return Promise.all(promises)
   })
 
-  afterAll(function (done) {
-    provider.removeInteractions().then(done, done.fail)
+  afterAll(function () {
+    return pactCleanup(provider)
```
Comment (HariBhandari07) [inline tests/appsTest.js:138]: Are we doing verification only from `pactHelper`?

File: tests/pactHelper.js [hunk 3]
```diff
@@ -513,6 +512,11 @@ function setGeneralInteractions (provider) {
   return promises
 }
 
+function pactCleanup (provider) {
```
Comment (individual-it) [inline tests/pactHelper.js:515]: maybe call it `verifyAndCleanup`

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/pactHelper.js (hunk 3 above)
Comment (dpakach) [reply tests/pactHelper.js:515]: @individual-it can we do that after all the related PRs get merged, I'll get conflicts on all other PRs because of this

**Review by individual-it — APPROVED** [review]

File: tests/pactHelper.js (hunk 3 above)
Comment (individual-it) [reply tests/pactHelper.js:515]: :+1:

**Pull request merged**